bottlerocket/agents/src/bin/eks-resource-agent/eks_provider.rs

```diff
@@ -128,6 +131,168 @@ enum ClusterConfig {
         region: String,
     },
 }
+#[derive(Serialize, Debug, EnumString)]
+enum IPFamily {
```

Comment:
I think the names of enums are supposed to be PascalCase which would make this IpFamily. If that is a conflict then I think it should be renamed to something else to reduce confusion.